bin/resources/GameIndex.yaml

```diff
@@ -2102,24 +2102,34 @@ SCAJ-30011:
     roundSprite: 1 # Fixes chromatic effect.
     autoFlush: 1 # Fixes sun occlusion.
 SCCS-40001:
-  name: "Ape Escape 2"
+  name: 捉猴啦 2
+  name-sort: "Zhuo Hou La 2"
+  name-en: "Ape Escape 2"
   region: "NTSC-C"
   gameFixes:
     - SoftwareRendererFMVHack # Fixes interlacing.
   gsHWFixes:
     mipmap: 2 # Fixes miptrick texture effects.
     trilinearFiltering: 1 # Fixes miptrick blending.
 SCCS-40002:
-  name: "Devil May Cry 2 - Disc 1"
+  name: 恶魔猎人2 (Disc 1)
+  name-sort: "'E'mo Lieren 2 (Disc 1)"
+  name-en: "Devil May Cry 2 - Disc 1"
   region: "NTSC-C"
 SCCS-40003:
-  name: "Devil May Cry 2 - Disc 2"
+  name: 恶魔猎人2 (Disc 2)
+  name-sort: "'E'mo Lieren 2 (Disc 2)"
+  name-en: "Devil May Cry 2 - Disc 2"
   region: "NTSC-C"
 SCCS-40004:
-  name: "XIGO - Zuihou de Touzi"
+  name: XIGO: 最后的骰子
```

Comment:
`XIGO：最后的骰子`

Reply:
it's probably upset that you used : and not -
you should replace that

Reply:
As far as I can tell, every reference to the Chinese title of the game use colon.

This should be a convention to use "additional names" when naming games in Chinese.

Reply:
You can't use colons in the names in the db so you have to use a dash.

Reply:
Even if the original title contains a colon?

Reply:
The gamedb isn't going to change it's mind :P It can't accept them because of how the filenames are used, change them to hyphens, please.

Reply:
Got it. Will fixed.

Reply:
Thank you, let's see if it passes now

Reply:
Sorry for the confusion, you can use colons if the original titles contain colons after #10494 is merged.

Reply:
What about cover images? They are one of the primary reasons why they weren't used as it will silent fail on this aspect.
![image](https://github.com/PCSX2/pcsx2/assets/24227051/e8f189f3-fb8d-4c4d-b1bf-97b415d5013c)

Reply:
Cover images should already be fine on master. The path is sanitized to replace any illegal characters with underscores.